webapp/src/serial.tsx

```diff
@@ -223,10 +223,10 @@ export class Editor extends srceditor.Editor {
                         </div>
                     </div>
                     <div className="rightHeader">
-                        <sui.Button class="ui icon circular small inverted button" ariaLabel={lf("Close")} onClick={this.goBack.bind(this)}>
-                            <sui.Icon icon="close" />
+                        <sui.Button class="ui icon circular small button closeEditor" ariaLabel={lf("Close")} onClick={this.goBack.bind(this)}>
```

Comment:
Should this be over to the left? Makes more sense to go back that way. Especially if you're using the arrow left icon.

Reply:
Just to the left of the download button?  Or *all* the way to the left, by the play/pause button?

Reply:
All the way to the left. Consistent with most back buttons